code/src/java/pcgen/core/Kit.java

```diff
@@ -375,8 +375,7 @@ public String getInfo(PlayerCharacter aPC)
 			info.append("  <b>Requirements</b>: ").append(aString);
 		}
 
-		List<BaseKit> sortedObjects = new ArrayList<>();
-		sortedObjects.addAll(getSafeListFor(ListKey.KIT_TASKS));
+		List<BaseKit> sortedObjects = new ArrayList<>(getSafeListFor(ListKey.KIT_TASKS));
 		Collections.sort(sortedObjects, new ObjectTypeComparator());
```

Comment:
Keep in mind, this clashes a bit with your other PR with the `list.sort()` calls.